### PR TITLE
Another clean up for global.mtls.enable

### PR DESCRIPTION
**Pull request by diemtvu**
Data plane security should be controlled by policy. We want to remove references to this values.global.mtls.enable option so it can be cleaned up in future release.

### CODE REVIEW

File: content/en/docs/reference/commands/istioctl/index.html [hunk 1]
```diff
@@ -3111,7 +3111,7 @@ <h2 id="istioctl-manifest-generate">istioctl manifest generate</h2>
 <td><code>-s</code></td>
 <td>Override an IstioOperator value, e.g. to choose a profile
 (--set profile=demo), enable or disable components (--set components.policy.enabled=true), or override Istio
-settings (--set values.global.mtls.enabled=true). See documentation for more info:
+settings (--set values.global.controlPlaneSecurityEnabled=true). See documentation for more info:
```
Comment (lei-tang) [inline content/en/docs/reference/commands/istioctl/index.html]: Does the change on this line mean that global.mtls.enabled has the same semantic as global.controlPlaneSecurityEnabled?

File: content/en/docs/setup/install/multicluster/shared-gateways/index.md [hunk 2]
```diff
@@ -180,7 +180,6 @@ This will be used to access pilot on `cluster1` securely using the ingress gatew
     $ CLUSTER_NAME=$(kubectl --context=$CTX_CLUSTER2 config view --minify=true -o jsonpath='{.clusters[].name}')
     $ istioctl manifest apply --context=$CTX_CLUSTER2 \
       --set profile=remote \
-      --set values.global.mtls.enabled=true \
```
Comment (lei-tang) [inline content/en/docs/setup/install/multicluster/shared-gateways/index.md]: Does the removal on this line mean that global.mtls.enabled is deprecated? Has the related code been removed from the Istio implementation?

File: content/en/docs/reference/commands/istioctl/index.html (hunk 1 above)
Comment (diemtvu) [reply content/en/docs/reference/commands/istioctl/index.html]: No. The example is just to show overring default value, so it doesn't matter.

File: content/en/docs/setup/install/multicluster/shared-gateways/index.md (hunk 2 above)
Comment (diemtvu) [reply content/en/docs/setup/install/multicluster/shared-gateways/index.md]: We still have that flag, but don't want users to continue to use that. We will remove it in future release.

File: content/en/docs/reference/commands/istioctl/index.html (hunk 1 above)
Comment (howardjohn) [reply content/en/docs/reference/commands/istioctl/index.html]: We are also getting rid of controlPlaneSecurityEnabled so we might want a different one here as well :slightly_smiling_face:

Comment (diemtvu) [reply content/en/docs/reference/commands/istioctl/index.html]: Ok :). Changed to `--set values.grafana.enabled=true`

**Review by lei-tang — APPROVED** [review]

File: content/en/docs/reference/commands/istioctl/index.html (hunk 1 above)
Comment (ericvn) [reply content/en/docs/reference/commands/istioctl/index.html]: I believe this file gets autogenerated (see the note at the top). Any changes here will likely be overwritten when the next merge is done. I believe you need to update this in the istio/istio repo. @snible can help identify the file(s) if a simple search doesn't find them.

Comment (diemtvu) [reply content/en/docs/reference/commands/istioctl/index.html]: Ah, thanks. Revert this change and create https://github.com/istio/istio/pull/21645

File: content/en/docs/setup/install/istioctl/index.md [hunk 3]
```diff
@@ -41,10 +41,10 @@ Kubernetes configuration. The `default` profile is a good starting point
 for establishing a production environment, unlike the larger `demo` profile that
 is intended for evaluating a broad set of Istio features.
 
-If you want to secure Istio control plane service endpoints on top of the `default` profile, you can set the security related configuration parameters:
+If you want to enable `grafana` dashboard on top of the `default` profile, you can set the `grafana` related configuration parameters:
 
 {{< text bash >}}
-$ istioctl manifest apply --set values.global.controlPlaneSecurityEnabled=true
+$ istioctl manifest apply --set values.grafana.enabled=true
```
Comment (richardwxn) [inline content/en/docs/setup/install/istioctl/index.md]: recommended way to enable grafana: `istioctl manifest apply --set addonComponents.grafana.enabled=true`

Comment (diemtvu) [reply content/en/docs/setup/install/istioctl/index.md]: done

File: content/en/docs/setup/install/istioctl/index.md [hunk 4]
```diff
@@ -41,10 +41,10 @@ Kubernetes configuration. The `default` profile is a good starting point
 for establishing a production environment, unlike the larger `demo` profile that
 is intended for evaluating a broad set of Istio features.
 
-If you want to secure Istio control plane service endpoints on top of the `default` profile, you can set the security related configuration parameters:
+If you want to enable `grafana` dashboard on top of the `default` profile, you can set the `grafana` related configuration parameters:
```
Comment (rcaballeromx) [inline content/en/docs/setup/install/istioctl/index.md]:
```suggestion
To enable the Grafana dashboard on top of the `default` profile, set the `addonComponents.grafana.enabled` configuration parameter with the following command:
```

File: content/en/docs/setup/install/multicluster/shared-vpn/index.md [hunk 5]
```diff
@@ -450,12 +450,11 @@ and endpoint to allow the remote sidecars to resolve the
     $ kubectl create secret generic cacerts -n istio-system --from-file=samples/certs/ca-cert.pem --from-file=samples/certs/ca-key.pem --from-file=samples/certs/root-cert.pem --from-file=samples/certs/cert-chain.pem
     {{< /text >}}
 
-1. Deploy the Istio control plane with security enabled for the control plane
-   and the application pod:
+1. Deploy the Istio control plane with security enabled for the control plane:
```
Comment (rcaballeromx) [inline content/en/docs/setup/install/multicluster/shared-vpn/index.md]:
```suggestion
1. Deploy the Istio control plane with its security features enabled:
```

File: content/en/docs/setup/install/multicluster/shared-vpn/index.md [hunk 6]
```diff
@@ -473,14 +472,14 @@ and endpoint to allow the remote sidecars to resolve the
    in the [setting environment variables section](#environment-var).
 
 1. The following command deploys the remote cluster's components with security
-   enabled for the control plane and the application pod and enables the
+   enabled for the control plane and enables the
```
Comment (rcaballeromx) [inline content/en/docs/setup/install/multicluster/shared-vpn/index.md]:
```suggestion
   features for the control plane enabled, and enables the
```

File: content/en/docs/tasks/security/authentication/authn-policy/index.md [hunk 7]
```diff
@@ -16,8 +16,12 @@ the underlying concepts in the [authentication overview](/docs/concepts/security
 * Understand Istio [authentication policy](/docs/concepts/security/#authentication-policies) and related
 [mutual TLS authentication](/docs/concepts/security/#mutual-tls-authentication) concepts.
 
-* Install Istio on a Kubernetes cluster with global mutual TLS disabled (e.g, use the demo configuration profile, as described in
-[installation steps](/docs/setup/getting-started), or set the `global.mtls.enabled` installation option to false).
+* Install Istio on a Kubernetes cluster with default profile, as described in
```
Comment (rcaballeromx) [inline content/en/docs/tasks/security/authentication/authn-policy/index.md]:
```suggestion
* Install Istio on a Kubernetes cluster with the `default` configuration profile, as described in
```

File: content/en/docs/tasks/security/citadel-config/plugin-ca-cert/index.md [hunk 8]
```diff
@@ -15,7 +15,9 @@ operator-specified root certificate. This task demonstrates an example to plug c
 
 ## Before you begin
 
-Follow the [Istio installation guide](/docs/setup/install/istioctl/) to install Istio with mutual TLS enabled.
+1. Follow the [Istio installation guide](/docs/setup/install/istioctl/) to install Istio.
+
+1. Enable mutual TLS in STRICT mode as show in [authentication policy](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode). This step is optional as the default mode (`PERMISSIVE`) already supports mutual TLS.
```
Comment (rcaballeromx) [inline content/en/docs/tasks/security/citadel-config/plugin-ca-cert/index.md]:
```suggestion
1. **Optional:** Enable mutual TLS in `STRICT` mode as show in the [authentication policy task](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode). The default `PERMISSIVE` mode already supports mutual TLS traffic.
```

File: content/en/docs/tasks/security/citadel-config/plugin-ca-cert/index.md [hunk 9]
```diff
@@ -48,11 +50,11 @@ The following steps enable plugging in the certificates and key into Citadel:
         --from-file=samples/certs/cert-chain.pem
     {{< /text >}}
 
-1.  Redeploy Citadel with `global.mtls.enabled` set to `true` and `security.selfSigned` to `false`.
+1.  Redeploy Citadel with `security.selfSigned` to `false`.
```
Comment (rcaballeromx) [inline content/en/docs/tasks/security/citadel-config/plugin-ca-cert/index.md]: Is this correct for 1.5? How is this working with Istiod?

**Review by rcaballeromx — CHANGES_REQUESTED** [review]

File: content/en/docs/tasks/security/citadel-config/plugin-ca-cert/index.md (hunk 9 above)
Comment (diemtvu) [reply content/en/docs/tasks/security/citadel-config/plugin-ca-cert/index.md]: https://github.com/istio/istio.io/pull/6644/files#diff-8 taking care of this, so I only remove global.mtls.enabled in this one for consistency.

Comment (rcaballeromx) [inline content/en/docs/tasks/security/citadel-config/plugin-ca-cert/index.md]:
```suggestion
1.  Redeploy Citadel with `security.selfSigned`set to `false`.
```

**Comment by rcaballeromx** [comment]
@diemtvu Only one verb missing, thanks.

**Review by rcaballeromx — APPROVED** [review]
All comments addressed, thanks.

**Pull request merged**